client/src/pages/Search/AddressSearch.js

```diff
@@ -2,8 +2,23 @@ import { IconButton, InputAdornment, TextField } from "@mui/material";
 import { usePlacesWidget } from "react-google-autocomplete";
 import { HUNTER_COLLEGE_ADDRESS } from "./constants";
 import ClearIcon from "@mui/icons-material/Clear";
+import getUserCoordinates from "../../utils/getUserCoordinates";
+import axios from "axios";
+import MyLocationIcon from "@mui/icons-material/MyLocation";
+import { useState } from "react";
 
 export default function AddressSearch({ updateFields }) {
+  const [address, setAddress] = useState(HUNTER_COLLEGE_ADDRESS);
+
+  async function handleClick() {
+    const coordinates = await getUserCoordinates();
+    const latitude = coordinates.coordinates.latitude;
+    const longitude = coordinates.coordinates.longitude;
+    const reversegeolocate = `https://maps.googleapis.com/maps/api/geocode/json?latlng=${latitude},${longitude}&key=${process.env.REACT_APP_GOOGLE_API_KEY}`;
```

Comment:
dont do this. This exposes our api key.
![image](https://github.com/csci-499-sp23/team-2-sp23/assets/51217000/6c4ba593-9f11-4478-aac7-8bcd07895b8b)

Reply:
ways we can resolve this:

1. Leave it, but restrict permissions based on domain and ip address
2. Let the server handle address match query